python-libraries/data-platform-catalogue/tests/client/datahub/test_datahub_client.py

```diff
@@ -481,40 +493,36 @@ def test_create_athena_database_and_table(
         """
         Case where we create separate database and table
         """
-        mock_graph = base_mock_graph
-        mock_graph.import_file(golden_file_in_db)
+        base_mock_graph.import_file(golden_file_in_db)
         with patch(
             "data_platform_catalogue.client.datahub.datahub_client.DataHubCatalogueClient.check_entity_exists_by_urn"
         ) as mock_exists:
             mock_exists.return_value = True
-            fqn_db = datahub_client.upsert_athena_database(metadata=database)
-            fqn_t = datahub_client.upsert_athena_table(metadata=table)
+            fqn_db = datahub_client.upsert_database(metadata=database)
+            fqn_t = datahub_client.upsert_table(metadata=table)
 
         fqn_db_out = "urn:li:container:my_database"
         assert fqn_db == fqn_db_out
 
-        fqn_t_out = (
-            "urn:li:dataset:(urn:li:dataPlatform:athena,my_database.my_table,PROD)"
-        )
+        fqn_t_out = "urn:li:dataset:(urn:li:dataPlatform:athena,database.Dataset,PROD)"
         assert fqn_t == fqn_t_out
 
         output_file = Path(tmp_path / "datahub_create_athena_table.json")
         base_mock_graph.sink_to_file(output_file)
         check_snapshot("datahub_create_athena_table.json", output_file)
 
-    def test_create_athena_table_with_metadata(
+    def test_create_table_with_metadata(
```

Comment:
This name isn't reflecting what the test does anymore

I think this one is testing the case where you create a table independently of a database, whereas the one above tests creating a database and table that relate to each other